File: 백준/Silver/11399. ATM/ATM.py
import sys

input = sys.stdin.readline

N = int(input())
P = sorted(map(int,input().split()))
ATM = [0] * N
ATM[0] = min(P)

for i in range(1,len(P)) :
    ATM[i] = ATM[i - 1] + P[i]

print(sum(ATM))

File: 백준/Silver/11399. ATM/ATM.js
const readLine = require('fs').readFileSync('dev/stdin').toString().trim();
const [N, times] = readLine.split('\n');
const convertTimes = times
    .split(' ')
    .map(Number)
    .sort((a, b) => a - b);
let waitTime = 0;
const minTime = convertTimes.reduce((acc, cur) => {
    acc += cur + waitTime;
    waitTime += cur;
    return acc;
}, 0);
console.log(minTime);